exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSystemPlugin.java

```diff
@@ -76,6 +79,16 @@ public FileSystemPlugin(FileSystemConfig config, DrillbitContext context, String
           fsConf.set(s, config.config.get(s));
         }
       }
+
+      logger.info("Original FileSystem default fs configuration {} {}",
+        fsConf.getTrimmed(FS_DEFAULT_NAME),
+        fsConf.getTrimmed(FileSystem.FS_DEFAULT_NAME_KEY));
+
+      if (logger.isInfoEnabled()) {
+        logger.info("Who made me? {}", new RuntimeException("Who made me?"));
```

Comment:
Is it proper message or you just forgot to delete it?

Reply:
removed